site/content/docs/4.3/examples/cover/index.html

```diff
@@ -3,33 +3,32 @@
 title: Cover Template
 extra_css:
   - "cover.css"
-body_class: "text-center"
+html_class: "h-100"
+body_class: "text-center text-white bg-dark d-flex h-100"
```

Comment:
```suggestion
body_class: "d-flex h-100 text-center text-white bg-dark"
```

I order utilities by our CSS property order whenever possible, so all the display/box-model stuff is at the front.